content/ai/concepts/search-algorithms/terms/breadth-first-search/breadth-first-search.md

```diff
@@ -0,0 +1,117 @@
+---
+Title: 'Breadth-First Search'
+Description: 'A shortest-path and traversing algorithm for unweighted graphs.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Graphs'
+  - 'Graph Search'
+  - 'Algorithms'
+CatalogContent:
+  - 'learn-python'
+  - 'paths/computer-science'
+---
+
+**Breadth-first search (BFS)** is a traversing algorithm for unweighted graphs. This is a foundational algorithm in [graph](https://www.codecademy.com/resources/docs/general/graph) theory from which many other algorithms start.
+
+## Features
+
+Some of the features and constraints that define the use and functionality of a breadth-first search algorithm include the following:
+
+- A relatively simple and efficient algorithm for finding the shortest-path in graphs that do not have edge weights.
+- It has a time complexity of O (vertices + edges).
```

Comment:
What do you think about putting the time complexity in back ticks? I'm realizing as is it kind of looks like the time complexity is "O".  
```suggestion
- It has a time complexity of `O (vertices + edges)`.
```

Reply:
I placed the term in italics and expanded the line a bit

Reply:
@caupolicandiaz I do like how it looks now, but do you know if we're able to use italics now? There is this old thread in Slack that mentioned staying away from italics because of the design of the system. Do you know if this has changed?